kernel/src/main/java/org/kframework/compile/checks/CheckAtt.java

```diff
@@ -145,7 +148,7 @@ private void check(Production prod) {
     }
 
     private void check(Rule rule) {
-        if (rule.isMacro()) {
+        if (rule.isMacro() && !backend.equals(Backends.JAVA)) {
```

Comment:
This isn't really a good solution here, the better solution is to migrate the parts of the tutorial that use the Java backend over to the Haskell backend. Would rather leave these warnings on, because they need to be fixed for those definitions to work on the haskell backend properly.

Reply:
And this is for macros too, so we shoudl probably just fix the definitions.

Reply:
Reverted that change.